Fix build on SLE12SP5 with gcc 4.8 that does not support this way of compound
intialization. RB_ROOT zeros the pointer, here we rely on the implicit zeroing
of static variables.

--- check/qgroup-verify.c.orig	2023-03-07 19:50:06.510639004 +0100
+++ check/qgroup-verify.c	2023-03-07 19:50:14.966639036 +0100
@@ -86,7 +86,7 @@
 	unsigned int		rescan_running:1;
 	unsigned int		qgroup_inconsist:1;
 	u64			scan_progress;
-} counts = { .root = RB_ROOT };
+} counts;
 
 static LIST_HEAD(bad_qgroups);
 
